src/clr-core/icon-shapes/collections/travel.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClarityIcons } from '../icon.service';
+import { IconAlias, IconShapeTuple } from '../interfaces/icon.interfaces';
+
+import { airplaneIcon, airplaneIconName } from '../shapes/airplane';
+import { bicycleIcon, bicycleIconName } from '../shapes/bicycle';
+import { boatIcon, boatIconName } from '../shapes/boat';
+import { campervanIcon, campervanIconName } from '../shapes/campervan';
+import { carIcon, carIconName } from '../shapes/car';
+import { caravanIcon, caravanIconName } from '../shapes/caravan';
+import { compassIcon, compassIconName } from '../shapes/compass';
+import { ferryIcon, ferryIconName } from '../shapes/ferry';
+import { mapIcon, mapIconName } from '../shapes/map';
+import { mapMarkerIcon, mapMarkerIconName } from '../shapes/map-marker';
+import { trailerIcon, trailerIconName } from '../shapes/trailer';
+import { truckIcon, truckIconName } from '../shapes/truck';
+
+export const travelCollectionIcons: IconShapeTuple[] = [
+  airplaneIcon,
+  bicycleIcon,
+  boatIcon,
+  carIcon,
+  campervanIcon,
+  caravanIcon,
+  compassIcon,
+  ferryIcon,
+  mapIcon,
+  mapMarkerIcon,
+  trailerIcon,
+  truckIcon,
+];
+
+export const travelCollectionAliases: IconAlias[] = [[airplaneIconName, ['plane']], [caravanIconName, ['auto']]];
+
+/**
+ * Function that can be called to load the core icon set.
+ *
+ * ```typescript
+ * import { loadTravelIconSet } from '@clr/core/icon';
+ *
+ * loadTravelIconSet();
```

Comment:
nit: sync the demo code in the tsdoc to the updated imports similar to the other collection funcs